api/api.go

```diff
@@ -207,6 +207,7 @@ func New(requiredUserAgent string, requiredPassword string, cs modules.Consensus
 		router.GET("/renter", api.renterHandlerGET)
 		router.POST("/renter", RequirePassword(api.renterHandlerPOST, requiredPassword))
 		router.GET("/renter/contracts", api.renterContractsHandler)
+		router.GET("/renter/contract/*contractid", api.renterContractDetailHandler)
```

Comment:
So: 

1. `api.RenterContract` needs to be extended. Is there a reason not to expose everything in `modules.RenterContract`?
0. @lukechampine We could make this the default behaviour and attach a field to `api.RenterContract` to indicate whether a contract is historical or current. Thoughts?